apps/src/applab/designMode.js

```diff
@@ -180,7 +192,9 @@ designMode.updateProperty = function(element, name, value) {
     case 'style-width':
       var newWidth = appendPx(value);
       element.style.width = newWidth;
-      element.parentNode.style.width = newWidth;
+      if (isDraggableContainer(element.parentNode)) {
+        element.parentNode.style.width = newWidth;
+      }
```

Comment:
I'm not sure I understand whether this was necessary before (and its absence was causing behavior) or why it has become necessary now. Would it make sense to do this for other properties such as width and height too?